functions/populateCounselors.ts

```diff
@@ -8,34 +8,45 @@ import {
 
 const TokenValidator = require('twilio-flex-token-validator').functionValidator;
 
-const send = (response: TwilioResponse) => (statusCode: number) => (body: string | object) => (
-  callback: ServerlessCallback,
+// TODO: Factor out into lib
+const send = (statusCode: number) => (body: string | object) => (callback: ServerlessCallback) => (
```

Comment:
Is it better to curry all the parameters vs have a list of them?

Reply:
Here the idea of currying is because we can have something like `const error404 = send(404)({some error message})` and then use it like `error404(callback)(response);` (taking advantage of partial application). But I'll change it to a listed version if you prefer!

Reply:
I see.  It's fine to have it this way, I just wanted to understand why.  Happy to keep as is given the description.